scripts/gha/test_simulator.py

```diff
@@ -517,7 +517,7 @@ def _create_and_boot_simulator(apple_platform, device_name, device_os):
         print("::warning ::Unable to find %s version %s, will fall back to %s" % (apple_platform, device_os, available_versions[-1]))
       device_os = available_versions[-1]
 
-    args = ["xcodes", "runtimes", "install", "%s %s" % (apple_platform, device_os)]
+    args = ["sudo", "xcodes", "runtimes", "install", "%s %s" % (apple_platform, device_os)]
```

Comment:
Update the info log message below to say it uses sudo; that way, if a user is running this on their machine, they won't be surprised by the password prompt.

Reply:
It prints out the args array, so the sudo should be included in the message

Reply:
I guess that's fine, though not quite as user-friendly as it could be. :P